consensus/polybft/contractsapi/helper.go

```diff
@@ -13,6 +14,16 @@ type StateTransactionInput interface {
 	DecodeAbi(b []byte) error
 }
 
+// Event is an abstraction for Ethereum events
+type Event interface {
```

Comment:
Is this used?

Reply:
I thought it would be useful, but eventually, it isn't. I''ve removed it: https://github.com/0xPolygon/polygon-edge/pull/1616/commits/3cf102887c9beb0522a907a7b290b832927d7915